src/components/ReportContact.jsx

```diff
@@ -1,83 +1,80 @@
 import React from 'react';
-import { withStyles } from '@material-ui/core/styles';
+import styled from 'styled-components';
 import { ArrowBack } from '@material-ui/icons';
-import Typography from '@material-ui/core/Typography';
-import Button from '@material-ui/core/Button';
 import PropTypes from 'prop-types';
 
+import { 
+  Typography,
+  Button,
+} from '@material-ui/core';
 
-const styles = {
-
-  container: {
-    display: 'flex',
-    alignItems: 'center',
-    marginBottom: 16,
-  },
-
-  button: {
-    minWidth: 46,
-    width: 46,
-    height: 46,
-    backgroundColor: '#46A440',
-    marginRight: 16,
-    color: 'white',
-  },
-
-  link: {
-    textDecoration: 'none',
-    color: 'white',
-  },
-
-  text: {
-    color: 'white',
-    fontSize: '1.7rem',
-  },
-
-  textSmall: {
-    color: 'white',
-    fontSize: '1.4rem',
-  },
-
-  ref: {
-    color: 'white',
-    textDecoration: 'none',
-    '&:hover': {
-      color: '#CACACA',
-    },
-  },
+const Container = styled.div`
+  display: flex';
+  align-items: center;
+  margin-bottom: 16px;
+`;
 
-};
+const ButtonStyled = styled(Button)`
+  && {
+  min-width: 46px;
+  width: 46px;
+  height: 46px;
+  background-color: #46A440;
+  margin-px;ight: 16px;
+  color: whitepx;
+  }
+`;
+
+const Text = styled(Typography)`
+  && {
+  color: white;
+  font-size: 1.7rem;
+  }
+`;
+
+const TextSmall = styled(Typography)`
+  color: white;
+  font-size: 1.4rem;
+`;
+
+const Refs = styled.a`
+  color: white;
+  text-decoration: none;
+   &:hover: 
+     {
+      color: #CACACA;
+     }
+`;
 
 const ReportContact = (props) => {
-  const { classes, changeView } = props;
+  const { changeView } = props;
 
   return (
     <React.Fragment>
-      <div className={classes.container}>
-        <Button
+      <Container>
+        <ButtonStyled
           variant="contained"
-          className={classes.button}
           onClick={() => changeView('home')}
         >
           <ArrowBack />
-        </Button>
-        <Typography className={classes.text}>
+        </ButtonStyled>
+        <Text>
           Contact person:
-        </Typography>
-      </div>
-      <Typography className={classes.textSmall}>
+        </Text>
+      </Container>
+      <TextSmall>
         Mr Walter Linnert
         <br />
-        <a href="mailto:walterl@capeaghulas.gov.za" className={classes.ref}>
+        <Refs href="mailto:walterl@capeaghulas.gov.za">
           walterl@capeaghulas.gov.za
           <br />
-        </a>
-        <a href="tel: +27284255500" className={classes.ref}>
+        </Refs>
+        <Refs href="tel: +27284255500">
           028 425 5500
           <br />
-        </a>
+        </Refs>
         Dirkie Uys Street, Bredasdorp, 7280
-      </Typography>
+      </TextSmall>
```

Comment:
It looks like this regresses the styling, which is probably not intended?

![image](https://user-images.githubusercontent.com/630271/56357436-0e2c6280-61dc-11e9-8e38-c3135f0a166b.png)

Reply:
Resolved from commit:
  6dd9a133ad15ae3fe6071282bfb896f1339b7770
Pushed to PR: 
      mish/SelectAction-to-styled-take-2 -> mish/SelectAction-to-styled-take-2